### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by dylanryder708**
Hi @PiperNeed4 could you please review my code!

### CODE REVIEW

File: prac_06/car.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    def __init__(self, name="", fuel=0):
+        """Initialise a Car instance.
+
+        name: name of car
```
Comment (PiperNeed4) [inline prac_06/car.py:8]: Should be a comment above the associated line of code rather then in the docstring

File: prac_06/car.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        return f"{self.name}, fuel={self.fuel}, odometer={self._odometer}"
+
+    def add_fuel(self, amount):
+        """Add amount to the car's fuel."""
+        self.fuel += amount
+
+    def drive(self, distance):
+        """Drive the car a given distance.
+
+        Drive given distance if car has enough fuel
```
Comment (PiperNeed4) [inline prac_06/car.py:26]: Should also be a comment above the associated line of code rather then in the docstring

File: prac_06/guitar.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+"""
+Start: 9:45
```
Comment (PiperNeed4) [inline prac_06/guitar.py:2]: The prac wanted an estimate of time to complete as well as the actual time to complete

File: prac_06/guitars.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
+        print(guitar_to_add, "added.")
+        name = input("Name: ")
+
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
```
Comment (PiperNeed4) [inline prac_06/guitars.py:18]: The provided lines were intended only for testing and were supposed to be commented out or removed in the final version

File: prac_06/guitars.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    if guitars:
+        print("These are my guitars:")
+        for i, guitar in enumerate(guitars, 1):
+            vintage_string = ""
```
Comment (PiperNeed4) [inline prac_06/guitars.py:24]: Would work better as an else statement for the below if statement

File: prac_06/programming_language.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
+"""
+Start: 9:05
```
Comment (PiperNeed4) [inline prac_06/programming_language.py:2]: Same as above applies, estimate needed

File: prac_06/used_cars.py [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+def main():
+    """Demo test code to show how to use car class."""
+    my_car = Car("My car", 180)
+    my_car.drive(30)
+    print(f"Car has fuel: {my_car.fuel}")
+    print(my_car)
+
+    limo = Car("Limo", 100)
+    limo.add_fuel(20)
+    print(limo.fuel)
```
Comment (PiperNeed4) [inline prac_06/used_cars.py:19]: Might benefit from being an f string with additional words similar to line 14

**Comment by dylanryder708** [comment]
Thank you!

**Pull request merged**